lib/supplejack_api/engine.rb

```diff
@@ -36,6 +36,9 @@ class Engine < ::Rails::Engine
 
     config.to_prepare do
       ApplicationController.helper(ActionView::Helpers::ApplicationHelper)
+      Dir.glob(Rails.root + 'app/decorators/**/*_decorator*.rb').each do |c|
+        require_dependency(c)
+      end
```

Comment:
what is the purpose of this?

Reply:
This allows the DNZ API to load the decorators within the DNZ API application. Happy to peer review this with you.